internal/host/validator.go

```diff
@@ -308,6 +312,26 @@ func (v *validator) hasMemoryForRole(c *validationContext) validationStatus {
 	}
 }
 
+func (v *validator) isValidPlatform(c *validationContext) validationStatus {
+	if c.inventory == nil {
+		return ValidationPending
+	}
+	return boolValue(!funk.ContainsString(invalidPlatforms, c.inventory.SystemVendor.ProductName))
```

Comment:
is it possible to have an empty system vendor?

Reply:
I was thinking about that, I don't think so,

Reply:
Depends on what "lshw" command provides for field "product"

Reply:
But we did see that sometimes because of a bug we don't get disks so why should it be different? 
@ronniel1 @avishayt what do you think?

Reply:
We should be defensive and check

Reply:
@avishayt so in case the vendor is not defined, what do we do?

Reply:
Unless we know exactly what platforms we support, I think that we can start collecting this information, maybe write it in an event or part of the metrics that we collect.

Reply:
@ErezAlster @nmagnezi Don't we already have this in metrics?

Reply:
@filanov  @avishayt I don't really see what is the difference, it's just a decision of what to do if we get an empty product field, no?

Reply:
I think that we should start with a black list. at least until we know all  out limitations.

Reply:
This is exactly what this PR is about.

Reply:
@oshercc `SystemVendor` is a pointer therefor it can be nill https://github.com/openshift/assisted-service/blob/master/models/inventory.go#L46

Reply:
> @ErezAlster @nmagnezi Don't we already have this in metrics?

Yes [we do](https://github.com/openshift/assisted-service/blob/eaefddf5e49302d2b0d4e353c88c0627debef821/internal/metrics/metricsManager.go#L216-L220). and the values default to "Unknown" in case we can't pull the naming for some reason.